src/settings-ui/Settings.UI/ViewModels/CmdNotFoundViewModel.cs

```diff
@@ -132,6 +137,11 @@ public bool IsArm64Arch
         public string RunPowerShellScript(string powershellExecutable, string powershellArguments, bool hidePowerShellWindow = false)
         {
             string outputLog = string.Empty;
+            if (isPowerShellPreviewDetected)
+            {
+                powershellExecutable = Path.Combine(powerShellPreviewPath, "pwsh-preview.cmd");
```

Comment:
This shouldn't be here. It should be the function that calls RunPowerShellScript that passes the right path to call. Or even a local variable of the view model that then gets set when detect preview and then gets passed by the caller.
Doing this change here is an internal hack.